packages/vite-plugin-svelte-inspector/package.json

```diff
@@ -44,7 +44,7 @@
     "debug": "^4.3.7"
   },
   "peerDependencies": {
-    "@sveltejs/vite-plugin-svelte": "^5.0.0-next.0 || ^5.0.0",
+    "@sveltejs/vite-plugin-svelte": "^5.0.0",
```

Comment:
I figured to update this along the way. I don't think it's a breaking change though removing prerelease version support.